ansible_shed/shed.py

```diff
@@ -40,14 +41,15 @@ def __init__(self, config_path: Path) -> None:
 
     def reload_config_vars(self) -> None:
         self.repo_path = Path(self.config[SHED_CONFIG_SECTION].get("repo_path"))
+        self.init_file = Path(f"{self.config[SHED_CONFIG_SECTION].get('repo_path')}/{self.config[SHED_CONFIG_SECTION]['ansible_playbook_init']}")
```

Comment:
Pathlib is made to do the path joining for you and then you get windows paths for free too if code is ran there. Pathlib is the lib for paths like ipaddress is for ip handling.
- Also, no need to use .get() here - None (which get defaults to) will error with the Path() constructor and I think KeyError is more revealing to the root cause rather than what ever error `Path(None)` raises.
```suggestion
        self.init_file = Path(
            self.config[SHED_CONFIG_SECTION]["repo_path"]) / self.config[SHED_CONFIG_SECTION]["ansible_playbook_init"]
        )
```